charts/topolvm/templates/controller/deployment.yaml

```diff
@@ -42,6 +42,7 @@ spec:
           image: "{{ .Values.image.repository }}:{{ default .Chart.AppVersion .Values.image.tag }}"
           command:
             - /topolvm-controller
+            - --leader-election-namespace={{ .Release.Namespace }}
```

Comment:
You said that not specifying a namespace means that the controller will use the namespace where the pod exists, so could you let me know why this is needed?

Reply:
I mainly did this to conform with the rest of the chart, because you also specified it for the sidecars. If this is not wanted I think we should think about removing it from the other sidecars as well. WDYT?

Reply:
That makes sense. I did not see this file entirely. LGTM!